notebooks/02.04-Getting-Spatial-Features-from-Pose.ipynb

```diff
@@ -73,7 +73,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "<img src=\"./Media/dihedral-final.png\" width=\"500\">"
+    "<img src=\"https://github.com/RosettaCommons/PyRosetta.notebooks/blob/master/notebooks/Media/dihedral-final.png?raw=true\" width=\"500\">"
```

Comment:
Why is this sourced from the web instead of locally?

Reply:
This method of displaying the image doesn't work in Colab. I changed it to a web address because it was the easiest way to fix it. I do know of a way to source it locally--let me know if that's preferable and I can change it to that